testing.md

```diff
@@ -30,18 +30,32 @@ You are free to create other testing environment configurations as necessary. Th
 
 ### Defining & Running Tests
 
-To create a test case, simply create a new test file in the `tests` directory. The test class should extend `TestCase`. You may then define test methods as you normally would using PHPUnit. To run your tests, simply execute the `phpunit` command from your terminal:
+To create a new test case, use the `make:test` Artisan command:
+
+    php artisan make:test FooTest
+
+This command will place a new `FooTest` class within your `tests` directory. You may then define test methods as you normally would using PHPUnit. To run your tests, simply execute the `phpunit` command from your terminal.
 
     <?php
 
+    use Illuminate\Foundation\Testing\WithoutMiddleware;
```

Comment:
That's what the default test case looks like. They are there for convenience so you can quickly use them if you want.